Monika After Story/game/script-holidays.rpy

```diff
@@ -2651,8 +2651,15 @@ label monika_aiwfc_song:
     $ disable_esc()
     $ mas_MUMURaiseShield()
 
-    $ play_song(None, 1.0)
+    # always unmute the music channel (or at least attempt to)
+    # TODO: there should probably be handling for sayori name case.
+    if songs.getVolume("music") == 0.0:
```

Comment:
I'm not sure that setting the user's volume is necessarily good practice as some people do keep it off for a reason (to avoid attention etc)

As well, the dialogue flow from the `monika_aiwfc` event somewhat leaves it in the player's court to do this themselves.

Reply:
this does not set user volume, it sets channel volume which is now completely not controlled by the user